README.md

```diff
@@ -51,60 +54,59 @@ In what follows assume that we have a clientService instance.
 
 ### Register the certificate
 Use the `registerCertificate` function to register a certificate on the Scalar DL network.
+```javascript
+await clientService.registerCertificate();
 ```
-const response = await clientService.registerCertificate();
-const status = response.getStatus();
-const message = response.getMessage();
-```
-Please refer to the [Status code](#status-code) section below for the details of status.
 
 ### Register contracts
 Use the `registerContract` function to register a contract.
-```
-const response = await clientService.registerContract('contractId', 'com.example.contract.contractName', contractUint8Array, propertiesObject);
-const status = response.getStatus();
-const message = response.getMessage();
+```javascript
+await clientService.registerContract('contractId', 'com.example.contract.contractName', contractUint8Array, propertiesObject);
 ```
 
 ### Register functions
 Use the `registerFunction` function to register a function.
-```
-const response = await clientService.registerFunction('functionId, 'com.example.function.functionName', functionUint8Array);
-const status = response.getStatus();
-const message = response.getMessage();
+```javascript
+await clientService.registerFunction('functionId, 'com.example.function.functionName', functionUint8Array);
 ```
 
 ### List registered contracts
 Use `listContracts` function to list all registered contracts.
-```
-const response = await clientService.listContracts();
-const message = response.getMessage();
-const contracts = JSON.parse(message);
+```javascript
+const constracts = await clientService.listContracts();
 ```
 
 ### Execute a contract
 Use `executeContract` function to execute a registered contract. It will also execute a function if `_functions_` is given in the argument.
-```
+```javascript
 const response = await clientService.executeContract('contractId', argumentObject);
```

Comment:
Sorry for saying this too late...
`executeContract` and `validateLedger` in Java SDK,  they don't return grpc responses as it is any more for better usability. They are wrapped with `ContractExecutionResult` and `LedgerValidationResult` now. Please check them for more detail. 
(for example, ContractExecutionResult uses JsonObject instead of string given from grpc)
Anyways, I would like other SDKs to work similarly to Java one.

Reply:
No problem.
Let me fix this in scalardl-javascript-sdk-base

Reply:
Thank you ! 🙇‍♂️